tidb-cloud/premium/connect-to-premium-via-aws-private-endpoint.md

```
@@ -115,34 +119,42 @@ To use the AWS Management Console to create a VPC interface endpoint, perform th
+    - In the **Create AWS Private Endpoint Connection** dialog after the private endpoint is created.
+    - On the **Networking** page of your instance. In the **AWS Private Endpoint** area, locate the private endpoint, and then click **...** > **Enable DNS**.
+
+2. Run the command in your AWS CLI.
 
 ```bash
-aws ec2 modify-vpc-endpoint --vpc-endpoint-id ${your_vpc_endpoint_id} --private-dns-enabled
+aws ec2 modify-vpc-endpoint --vpc-endpoint-id ${your_vpc_endpoint_id} --region ${your_region} --private-dns-enabled
 ```
 
-Alternatively, you can find the command on the **Networking** page of your instance. Locate the private endpoint and click **...** > **Enable DNS** in the **Action** column.
+3. After the command is executed successfully, return to the TiDB Cloud dialog and click **Done**.
```

markdownlint-cli2 (0.23.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/2/3

(MD029, ol-prefix)